lib/boilerplate.js

```diff
@@ -101,7 +110,10 @@ const boilerplate = function (gulp, opts) {
   }
 
   if (opts.watch) {
-    const watchSequence = opts.eslintOnWatch ?
+    if (opts.eslintOnWatch) {
+      console.log(`The 'eslintOnWatch' option is deprecated. Use 'lintOnWatch' instead`); // eslint-disable-line no-console
+    }
+    const watchSequence = (opts.eslintOnWatch || opts.lintOnWatch) ?
```

Comment:
To make removing the deprecated `eslintOnWatch`, why not set `lintOnWatch` if `eslintOnWatch` is set, and then the rest of the logic can continue as if `eslintOnWatch` didn't exist.

Reply:
👍